compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/SymbolEnter.java

```diff
@@ -1159,16 +1159,21 @@ public void initPredeclaredModules(Map<Name, BPackageSymbol> predeclaredModules,
                         new Name(compUnits.get(index++).name)));
```

Comment:
This is the existing code. if comfortable shall we replace this to iterate over the entry set instead of keySet. We can avoid the call to `predeclaredModules.get(alias)` to avoid the lookup time. For a hashmap, this is not much but it is good practice to write a loop like this. Choose some variable names.

```
        for (Map.Entry<Name, BPackageSymbol> e : predeclaredModules.entrySet()) {
            Name alias = e.getKey();
```
So instead of `predeclaredModules.get(alias)` we can probably have something like.
```
e.getValue()
```

Reply:
I will add that.